README.md

```diff
@@ -187,3 +196,253 @@ Here, you will find various usage examples of ggbridge, each example provides a
 | --------------------------------------------- | --------------------------------------------- |
 | [2-way-tunneling](./examples/2-way-tunneling) | Enable client-to-server tunnels               |
 | [ggscout](./examples/ggscout)                 | Connect ggscout with the GitGuardian platform |
+
+
+## Troubleshooting
+### Debug image
+
+GGBridge provides two types of images for different use cases:
+
+- **Production image**: `ghcr.io/gitguardian/ggbridge:1.3.0` - Minimal, secure image without shell access
+- **Debug image**: `ghcr.io/gitguardian/ggbridge:1.3.0-shell` - Includes debugging tools
+
+**Available debug tools**: `bash`, `curl`, `net-tools`, `bind-tools`, `openssl`, `dig`, `nslookup`
+
+**How to switch to debug image**:
+- **Docker Compose**: Update the image tag in `docker-compose.yaml`
+- **Helm**: Update the image tag in `values.yaml`
+
+### Connectivity Tests
+
+#### 1. Client-Side Healthcheck
+
+Verify basic connectivity from the client to the server:
+
+```bash
+kubectl exec -it <ggbridge-client-pod> -- bash -c "curl http://127.0.0.1:9081/healthz"
```

Comment:
nit: prefer `$ggbridge_pod` for syntax, easier to read and apply, preventing the shell to interpret `<>` as redirects